src/axom/inlet/CMakeLists.txt

```diff
@@ -64,6 +64,9 @@ blt_add_library( NAME        inlet
                  FOLDER      axom/inlet
                  OBJECT      TRUE )
 
+# Set file back to C++ due to nvcc compiler error
+set_source_files_properties(LuaReader.cpp PROPERTIES LANGUAGE CXX)
```

Comment:
I only turned swapped problematic files back to C++ from CUDA. We may want to be more generous with doing this due to how slow nvcc is.